js/task-02/ajax.js

```diff
@@ -0,0 +1,34 @@
+function onSubmit(){
+  
+  const url = "./ajax-handler.php";
+	
+  var callback = function (text) {
+  console.log(text);
+  };
+
+  let username = document.getElementById("username").value;
+  let pass = document.getElementById("pass").value;
+  let pass2 = document.getElementById("pass2").value;
+
+  let obj = { username, pass, pass2 };
+
+  let jsonToBeSent = JSON.stringify(obj);
+	
+  ajax(url, { success: callback, 
```

Comment:
The request is sent even with invalid data. Please add the validations now as we have implemented the core functionality of the task.